examples/complete/main.tf

```diff
@@ -30,36 +18,11 @@ module "s3_bucket" {
   s3_replication_enabled       = local.replication_enabled
   s3_replica_bucket_arn        = join("", module.s3_bucket_replication_target.*.bucket_arn)
   s3_replication_rules         = local.s3_replication_rules
+  privileged_principal_arns = var.privileged_principal_enabled ? {
+    (local.principal_names[0]) = [""]
+    (local.principal_names[1]) = ["a/", "b/"]
+  } : {}
```

Comment:
in many cases, maps don't work in the ternary operator since TF wants the map and all items in the map to have the same type

Reply:
what is `["a/", "b/"]`?

Reply:
> what is `["a/", "b/"]`?
S3 bucket prefixes that the principal is limited to.

Reply:
Renamed